client/components/AuthForm.js

```diff
@@ -23,14 +25,24 @@ const AuthForm = props => {
           </label>
           <input name="password" type="password" />
         </div>
+        {location.pathname === "/signup" ? (
+          <div>
+            <label htmlFor="email">
+              <small>Email</small>
+            </label>
+            <input name="email" type="text" />
+          </div>
+        ) : (
+          <></>
```

Comment:
You can clean this up a little by changing the ternary to a straight boolean conditional 
```javacript
{location.pathname === '/signup' && <div>...</div>}
```